scanner/testdata/azure-pipelines-1.yml

```diff
@@ -7,3 +7,4 @@ steps:
     echo "Hello, pr!"
 - bash: |
     curl $(URL) | bash
+- script: npm install
```

Comment:
Would be good to add another test case which does not even have “pr:”
It’s more about parsing but just better have all the right assumptions in test files. Like a pipeline just on push to main with “npm install”